cellprofiler/modules/correctilluminationcalculate.py

```diff
@@ -206,209 +226,222 @@ def create_settings(self):
+      -  Find the convex hull that encloses those pixels
+      -  Set the intensity of the output image within the convex hull to
+         the current intensity
+    
+   The Convex Hull method can be used on an image whose objects are
+   darker than their background and whose illumination intensity
+   decreases monotonically from the brightest point.
+    
+**References**
+    
+-  J Lindblad and E Bengtsson (2001) “A comparison of methods for
+   estimation of intensity nonuniformities in 2D and 3D microscope
+   images of fluorescence stained cells.”, Proceedings of the 12th
+   Scandinavian Conference on Image Analysis (SCIA), pp. 264-271
+""" % globals())
 
         self.automatic_object_width = cps.Choice(
-                "Method to calculate smoothing filter size",
-                [FI_AUTOMATIC, FI_OBJECT_SIZE, FI_MANUALLY], doc='''
-            <i>(Used only if a smoothing method other than Fit Polynomial is selected)</i><br>
-            Calculate the smoothing filter size. There are three options:
-            <ul>
-            <li><i>%(FI_AUTOMATIC)s:</i> The size is computed as 1/40 the size of the image or
-            30 pixels, whichever is smaller.</li>
-            <li><i>%(FI_OBJECT_SIZE)s:</i> The size is obtained relative to the width
-            of artifacts to be smoothed.</li>
-            <li><i>%(FI_MANUALLY)s:</i> Use a manually entered value.</li>
-            </ul>''' % globals())
+            "Method to calculate smoothing filter size",
+            [FI_AUTOMATIC, FI_OBJECT_SIZE, FI_MANUALLY], doc="""\               
+*(Used only if a smoothing method other than Fit Polynomial is selected)*
+Calculate the smoothing filter size. There are three options:
+   
+-  *%(FI_AUTOMATIC)s:* The size is computed as 1/40 the size of the
+   image or 30 pixels, whichever is smaller.
+-  *%(FI_OBJECT_SIZE)s:* The size is obtained relative to the width of
+   artifacts to be smoothed.
+-  *%(FI_MANUALLY)s:* Use a manually entered value.
+""" % globals())
 
         self.object_width = cps.Integer(
-                "Approximate object size", 10, doc='''
-            <i>(Used only if %(FI_AUTOMATIC)s is selected for smoothing filter size calculation)</i><br>
-            Enter the approximate width of the artifacts to be smoothed, in pixels.''' % globals())
+            "Approximate object size", 10, doc="""\
+*(Used only if %(FI_AUTOMATIC)s is selected for smoothing filter size calculation)*
+Enter the approximate width of the artifacts to be smoothed, in pixels.
+""" % globals())
 
         self.size_of_smoothing_filter = cps.Integer(
-                "Smoothing filter size", 10, doc='''
-            <i>(Used only if %(FI_MANUALLY)s is selected for smoothing filter size calculation)</i><br>
-            Enter the size of the desired smoothing filter, in pixels.''' % globals())
+            "Smoothing filter size", 10, doc="""\
+*(Used only if %(FI_MANUALLY)s is selected for smoothing filter size calculation)*
```

Comment:
Add a newline between `*(Used only if...)*` and the next line. This will insert a line break in the rendered documentation. Otherwise, the text appears on the same line.